--- a/Makefile.am	2023-06-20 13:39:48.371700600 +0800
+++ b/Makefile.am	2023-06-20 14:16:19.773764000 +0800
@@ -20,23 +20,23 @@
 libtpm2_la_LDFLAGS= -no-undefined -avoid-version
 libtpm2_la_LIBADD=${DEPS_LIBS}
 libtpm2_la_SOURCES=e_tpm2.c e_tpm2-rsa.c e_tpm2-ecc.c tpm2-common.c
-libtpm2_la_CFLAGS=${DEPS_CFLAGS} -g -Werror
+libtpm2_la_CFLAGS=${DEPS_CFLAGS} -g
 
 create_tpm2_key_SOURCES=create_tpm2_key.c tpm2-common.c
 create_tpm2_key_LDADD=${DEPS_LIBS}
-create_tpm2_key_CFLAGS=${DEPS_CFLAGS} -Werror
+create_tpm2_key_CFLAGS=${DEPS_CFLAGS}
 
 load_tpm2_key_SOURCES=load_tpm2_key.c tpm2-common.c
 load_tpm2_key_LDADD=${DEPS_LIBS}
-load_tpm2_key_CFLAGS=${DEPS_CFLAGS} -Werror
+load_tpm2_key_CFLAGS=${DEPS_CFLAGS}
 
 seal_tpm2_data_SOURCES=seal_tpm2_data.c tpm2-common.c
 seal_tpm2_data_LDADD=${DEPS_LIBS}
-seal_tpm2_data_CFLAGS=${DEPS_CFLAGS} -Werror
+seal_tpm2_data_CFLAGS=${DEPS_CFLAGS}
 
 unseal_tpm2_data_SOURCES=unseal_tpm2_data.c tpm2-common.c
 unseal_tpm2_data_LDADD=${DEPS_LIBS}
-unseal_tpm2_data_CFLAGS=${DEPS_CFLAGS} -Werror
+unseal_tpm2_data_CFLAGS=${DEPS_CFLAGS}
 
 $(builddir)/%.1: $(srcdir)/%.1.in $(top_builddir)/%
 	$(HELP2MAN) --no-info -i $< -o $@ $(top_builddir)/$*

--- a/configure.ac	2023-06-20 13:39:48.379865300 +0800
+++ b/configure.ac	2023-06-20 14:08:54.564793600 +0800
@@ -78,7 +78,7 @@
 		  #endif
 		]]
 		)])
-		gcc $CFLAGS conftest.c -lcrypto
+		$CC $CFLAGS conftest.c -lcrypto
 		enginesdir=`./a.out|sed 's/ENGINESDIR: //'`
 	]
 	)

--- a/create_tpm2_key.c	2020-07-02 22:14:25.000000000 +0800
+++ b/create_tpm2_key.c	2023-06-20 13:45:30.264518200 +0800
@@ -22,7 +22,7 @@
 #include <openssl/pkcs12.h>
 #include <openssl/rand.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/e_tpm2-ecc.c	2020-07-02 22:14:25.000000000 +0800
+++ b/e_tpm2-ecc.c	2023-06-20 13:45:59.335124200 +0800
@@ -16,7 +16,7 @@
 #include <openssl/sha.h>
 #include <openssl/bn.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/e_tpm2-rsa.c	2020-07-02 22:14:25.000000000 +0800
+++ b/e_tpm2-rsa.c	2023-06-20 13:46:09.849810900 +0800
@@ -18,7 +18,7 @@
 #include <openssl/pem.h>
 #include <openssl/x509.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/e_tpm2.c	2020-07-02 22:14:25.000000000 +0800
+++ b/e_tpm2.c	2023-06-20 13:46:21.741078600 +0800
@@ -15,7 +15,7 @@
 #include <openssl/evp.h>
 #include <openssl/pem.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/load_tpm2_key.c	2020-07-02 22:14:25.000000000 +0800
+++ b/load_tpm2_key.c	2023-06-20 13:45:41.719382300 +0800
@@ -26,7 +26,7 @@
 #include <openssl/err.h>
 #include <openssl/rand.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/seal_tpm2_data.c	2020-07-02 22:14:25.000000000 +0800
+++ b/seal_tpm2_data.c	2023-06-20 13:44:39.093947000 +0800
@@ -16,7 +16,7 @@
 #include <openssl/pem.h>
 #include <openssl/ui.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/tpm2-common.c	2020-07-02 22:14:25.000000000 +0800
+++ b/tpm2-common.c	2023-06-20 13:44:30.422231600 +0800
@@ -22,7 +22,7 @@
 #include <openssl/err.h>
 #include <openssl/ui.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssresponsecode.h)
 #include TSSINCLUDE(tssmarshal.h)

--- a/unseal_tpm2_data.c	2020-07-02 22:14:25.000000000 +0800
+++ b/unseal_tpm2_data.c	2023-06-20 13:45:14.644606800 +0800
@@ -16,7 +16,7 @@
 #include <openssl/pem.h>
 #include <openssl/ui.h>
 
-#define TSSINCLUDE(x) < TSS_INCLUDE/x >
+#define TSSINCLUDE(x) <TSS_INCLUDE/x>
 #include TSSINCLUDE(tss.h)
 #include TSSINCLUDE(tssutils.h)
 #include TSSINCLUDE(tssmarshal.h)
